clients/githubrepo/graphql.go

```diff
@@ -205,19 +212,42 @@ func (handler *graphqlHandler) setup() error {
 			"issueCommentsToAnalyze": githubv4.Int(issueCommentsToAnalyze),
 			"reviewsToAnalyze":       githubv4.Int(reviewsToAnalyze),
 			"labelsToAnalyze":        githubv4.Int(labelsToAnalyze),
-			"commitsToAnalyze":       githubv4.Int(commitsToAnalyze),
+			"commitsToAnalyze":       githubv4.Int(CommitsToAnalyze),
 			"commitExpression":       githubv4.String(commitExpression),
+			"historyCursor":          (*githubv4.String)(nil),
 		}
-		if err := handler.client.Query(handler.ctx, handler.data, vars); err != nil {
-			handler.errSetup = sce.WithMessage(sce.ErrScorecardInternal, fmt.Sprintf("githubv4.Query: %v", err))
+		if CommitsToAnalyze < 99 {
```

Comment:
maybe document why 99 is a special number?

Reply:
Does this work `// if --number-of-commits flag set to greater than 99 we need to page to query the github api.`?

Reply:
SGTM. Maybe say that "graphQL API needs paging for results with size greater than 99"? Not clear whether "we need" is us for our implementation, or a requirement from graphQL